com.github.TheThirdOne.rars.metainfo.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be submitted upstream.

Reply:
The upstream source is [only distributed as a `.jar` file](https://github.com/TheThirdOne/rars#download), which uses the JVM to run (I don't believe AppStream is compatible with JAR files).